README.md

```diff
@@ -59,12 +60,17 @@ It include next modules:
 
 ---
 
+## About the project
+- You can read more about the project's technologies and modules description  in   <a href="https://github.com/ita-social-projects/Home/tree/dev/home-docs" target="_blank">home-docs</a> package.
+
 ## Installation
 
 - All the `code` required to get started
 - Images of what it should look like
 
 ### Required to install
+- Java 11
```

Comment:
Should we add maven to the list of required software?

Reply:
Yes, we should.